### ISSUE TITLE
PTT killing JVM prevents some files from being uploaded

### ISSUE THREAD

**Issue by alittley** [hashgraph/hedera-services#11365]
## Affected Tests

- `VMerkleDual-NDReconnect-LegacyIntake-500-60m`
- `VMerkleDual-NDReconnect-500-60m`

## Behavior

When the bad behavior occurs, all validators fail with message:

```
java.io.FileNotFoundException: /runner/_work/hedera-services/hedera-services/platform-sdk/regression/assets/results/4N/LegacyIntakePipeline/20240203-110236-GCP-LegacyIntakePipeline-4N/VMerkleDual-NDReconnect-LegacyIntake-500-60m/node0003/swirlds.json (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)
	at com.swirlds.fsts.validator.file.loader.ResultFileLoader.getFileStream(ResultFileLoader.java:115)
	at com.swirlds.fsts.workflow.Test.validate(Test.java:301)
	at com.swirlds.fsts.workflow.Test.validate(Test.java:277)
	at com.swirlds.fsts.workflow.TestFlow.validation(TestFlow.java:527)
	at com.swirlds.fsts.workflow.TestFlow.dispatch(TestFlow.java:196)
	at com.swirlds.fsts.workflow.Regression.testExecution(Regression.java:362)
	at com.swirlds.fsts.workflow.Regression.execute(Regression.java:256)
	at com.swirlds.fsts.cli.MainCommand.executeRegressionFlow(MainCommand.java:297)
	at com.swirlds.fsts.cli.MainCommand.call(MainCommand.java:285)
	at com.swirlds.fsts.cli.MainCommand.call(MainCommand.java:60)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at com.swirlds.fsts.Main.main(Main.java:31)
```

The way that PTT is trying to kill the JVM may be what is causing problems:

```
2024-02-03 12:46:03.896 28863    INFO  DEMO_QUORUM      <<scheduler signedStateFileManager>> PlatformTestingToolMain: Terminating the JVM [ consensusTime = 2024-02-03T12:44:27.606541314Z ]
2024-02-03 12:46:03.898 28864    INFO  STARTUP          <<scheduler signedStateFileManager>> SystemExitUtils: System exit requested (NO_ERROR)
thread requesting exit: <scheduler signedStateFileManager>
com.swirlds.platform.system.SystemExitUtils.exitSystem(SystemExitUtils.java:51)
	at com.swirlds.platform.system.SystemExitUtils.exitSystem(SystemExitUtils.java:87)
	at com.swirlds.demo.platform.PlatformTestingToolMain.logSuccessMessageAndFinishTest(PlatformTestingToolMain.java:1210)
	at com.swirlds.demo.platform.PlatformTestingToolMain.lambda$registerFinishAfterSaveStateDuringFreezeListener$19(PlatformTestingToolMain.java:1064)
	at com.swirlds.common.notification.internal.Dispatcher.handleDispatch(Dispatcher.java:142)
	at com.swirlds.common.notification.internal.Dispatcher.notifySync(Dispatcher.java:113)
	at com.swirlds.common.notification.internal.AsyncNotificationEngine.lambda$dispatch$0(AsyncNotificationEngine.java:99)
	at com.swirlds.common.notification.internal.AsyncNotificationEngine.invokeWithDispatcher(AsyncNotificationEngine.java:189)
	at com.swirlds.common.notification.internal.AsyncNotificationEngine.dispatch(AsyncNotificationEngine.java:93)
	at com.swirlds.common.notification.NotificationEngine.dispatch(NotificationEngine.java:76)
	at com.swirlds.platform.components.appcomm.AppCommunicationComponent.stateSavedToDisk(AppCommunicationComponent.java:95)
	at com.swirlds.common.wiring.wires.output.StandardOutputWire.forward(StandardOutputWire.java:68)
	at com.swirlds.common.wiring.schedulers.TaskScheduler.forward(TaskScheduler.java:259)
	at com.swirlds.common.wiring.wires.input.BindableInputWire.lambda$bind$0(BindableInputWire.java:76)
	at com.swirlds.common.wiring.schedulers.internal.SequentialThreadTask.handle(SequentialThreadTask.java:34)
	at com.swirlds.common.wiring.schedulers.internal.SequentialThreadTaskScheduler.run(SequentialThreadTaskScheduler.java:193)
	at java.lang.Thread.run(Thread.java:1583)
```

Examples:
- http://35.247.76.217:8095/swirlds-automation/release/0.46/4N/Reconnect/20240131-091638-GCP-Daily-Reconnect-4N/VMerkleDual-NDReconnect-500-60m/
- http://35.247.76.217:8095/swirlds-automation/release/0.46/4N/Reconnect/20240131-194633-GCP-Daily-Reconnect-4N/VMerkleDual-NDReconnect-500-60m/
- http://35.247.76.217:8095/swirlds-automation/release/0.46/4N/LegacyIntakePipeline/20240201-111700-GCP-LegacyIntakePipeline-4N/VMerkleDual-NDReconnect-LegacyIntake-500-60m/
- http://35.247.76.217:8095/swirlds-automation/release/0.46/4N/LegacyIntakePipeline/20240203-110236-GCP-LegacyIntakePipeline-4N/VMerkleDual-NDReconnect-LegacyIntake-500-60m/

**Comment by JeffreyDallas** [comment]
TLDR: test moved to validation stage even before the download file finished unzipped
yet (when the file is hug).

====

After JRS downloaded test result to the control box,
it called tar to unzip the test result file, but default wait time is only two minutes.


```
                        node.exec(
                                downloadCmd,
                                "Pack every file inside the working folder",
                                Duration.ofMinutes(ZIPPING_WAIT_MINUTES));

                        node.fileSystem().copyTo(packedFile, controlNode, currentNodeFolderPath);

                        node.exec(
                                new String[] {
                                    "sudo",
                                    "rm",
                                    "-f", // if file does not exist for deletion, don't throw error
                                    FilePathNormalizer.normalize(packedFile.toString())
                                },
                                "Remove the packed file, since we already have a local copy.");

                        final File localPackedFile = new File(currentNodeFolderPath, packedFile.getName());

                        controlNode.exec(
                                new String[] {
                                    "tar",
                                    "-xzf",
                                    localPackedFile.getAbsolutePath(),
                                    "--directory",
                                    currentNodeFolderPath.toString()
                                },
                                "Unpack the downloaded file with all files from the working folder");

```

```java
        try {
            final Process start = new ProcessBuilder(cmd).start();

            final ControlCommand controlCommand = new ControlCommand(start);
            if (waitTime.isZero()) {
                return controlCommand;
            }
            controlCommand.join(waitTime);
            return controlCommand;
```

When the file is too large, not being totally unzipped withinin two minutes, cause
the runner move to the next validation stage

```
2024-02-03 13:38:55.492 10823    DEBUG REMOTE_OPERATION << NodeManager #81 >> SSHConnection: SSH Command Execution [ remoteIpAddress = '35.222.26.128', command = 'sudo rm -f /opt/hgcapp/services-hedera/workingFolder.tar.gz' ]
2024-02-03 13:40:57.900 10826    TRACE POST_PROCESSING  <main> TestFlow: Result files copied to /runner/_work/hedera-services/hedera-services/platform-sdk/regression/assets/results/4N/LegacyIntakePipeline/20240203-110236-GCP-LegacyIntakePipeline-4N/VMerkleDual-NDReconnect-LegacyIntake-500-60m
2024-02-03 13:40:57.904 10829    INFO  TEST_EXECUTION   <main> TestFlow: Finished Test Post-Processing
2024-02-03 13:40:57.904 10832    INFO  TEST_EXECUTION   <main> TestFlow: Started Test Validation
```

That's why validator could not see any files, but later those files become available and uploaded to GCP storage
as expected.
